src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menFiltered = people.filter(i => i.sex === 'm');
```

Comment:
don't use letters for variables, use more specific names

```suggestion
  let menFiltered = people.filter(person => person.sex === 'm');
```